src/features/room/list/ui/HomeRoomStage.module.css

```
@@ -0,0 +1,143 @@
+.slot[data-slot="off-right"] .slotCard {
+  opacity: 0;
+  filter: brightness(0.72);
+}
+
+@media (max-width: 900px) {
+  .viewport {
+    --card-current-width: clamp(14rem, 44vw, 18rem);
+    --card-current-height: clamp(14rem, 44vw, 18rem);
+    --card-side-size: clamp(8.75rem, 24vw, 10.75rem);
+    --side-offset: clamp(10rem, 28vw, 14rem);
+    padding-bottom: 2rem;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)